### PR TITLE
docs: add Flow documentation

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

File: packages/lab/src/components/Flow/stories/Usage.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,427 @@
+import { Meta, Canvas, Story } from "@storybook/addon-docs";
+
+<Meta title="Lab/Flow/Usage" />
+
```
Comment (MEsteves22) [inline packages/lab/src/components/Flow/stories/Usage.stories.mdx:4]: I think we should add a title to this page 🙏 

Also, I believe we should make a disclaimer saying that this page maybe have breaking changes over time. We already have this on the component's page but it doesn't hurt saying it again.

File: packages/lab/src/components/Flow/stories/Usage.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+### Background <a id="background" />
+
+To add a background to the flow you should use the `<HvFlowBackground />` component. This is a wrapper around the
+[Background](https://reactflow.dev/docs/api/plugin-components/background/) component of `reactflow` and has the same API.
+
+### Controls <a id="controls" />
+
+To add the controls panel to the flow you should use the `<HvFlowControls />` component. This, again, uses the
+[Controls](https://reactflow.dev/docs/api/plugin-components/controls/) component from `reactflow` but you should check [our own
+documentation](?path=/docs/lab-flow--docs) for the `HvFlowControls` component also.
```
Comment (MEsteves22) [inline packages/lab/src/components/Flow/stories/Usage.stories.mdx:107]: This link is not working. I think it should be `./?path=/docs/lab-flow--docs` if I'm not mistaken.

File: packages/lab/src/components/Flow/stories/Usage.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,427 @@
...
+      {...props}
+    >
+      <div>Node content</div>
+    </HvFlowNode>
+  );
+};
+
+CustomNode.meta = {
+  label: "CustomNode",
+  groupId: "someNodeGroup",
```
Comment (MEsteves22) [inline packages/lab/src/components/Flow/stories/Usage.stories.mdx:146]: Maybe we should continue using the `groupId`'s defined earlier just for consistency and not create new groups.

File: packages/lab/src/components/Flow/stories/Usage.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,427 @@
...
+Whenever a change is made on a parameter while using the flow, that change is saved on the node's `data` property in the `reactflow` instance.
+It's saved in the form of:
+
+```typescript
+data: {
+  [paramterId]: [parameterValue]
+}
+```
+
+Current parameter types supported by the `HvFlow` component:
```
Comment (MEsteves22) [inline packages/lab/src/components/Flow/stories/Usage.stories.mdx:200]: Maybe we should specify the types (TS) for these parameters.

File: packages/lab/src/components/Flow/stories/Usage.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,427 @@
...
+}
+```
+
+While this hook can be updated to include more useful information, we advise you to use the `reactflow` instance directly if you want to do more
+advanced operations. Please check the [React Flow documentation](https://reactflow.dev/docs/api/hooks/use-react-flow/) to get more information,
+specially on the hooks section.
+
+# Custom Node Example <a id="custom-node-example" />
+
+Below you can find a full example of a custom Node that exercises that topics we talked about before:
```
Comment (MEsteves22) [inline packages/lab/src/components/Flow/stories/Usage.stories.mdx:313]: Typo: "that exercises that topics" => "that exercises the topics"

**Review by MEsteves22 — COMMENTED** [review]
LGTM 🙏 

Just some notes:

**Review by francisco-guilherme — APPROVED** [review]

**Pull request merged**